nthuoj/urls.py

```diff
@@ -2,7 +2,11 @@
 from django.contrib import admin
 
 urlpatterns = patterns('',
+    # Examples:
+    # url(r'^$', 'forgroup.views.home', name='home'),
+    # url(r'^blog/', include('blog.urls')),
+    url(r'^$', 'group.views.index'),
+    url(r'^list/$', 'group.views.list'),
```

Comment:
I think it's better to use groupList and groupDetail(next line). It can avoid confusing users.